vpr/src/place/place.cpp

```diff
@@ -1861,31 +2221,21 @@ static void revert_td_cost(const t_pl_blocks_to_be_moved& blocks_affected) {
 //
 //Relies on proposed_connection_delay and connection_delay to detect
 //which connections have actually had their delay changed.
-static void invalidate_affected_connection_delays(const t_pl_blocks_to_be_moved& blocks_affected,
+static void invalidate_affected_connection_delays(const std::vector<ClusterPinId>& sink_pins_affected,
                                                   ClusteredPinTimingInvalidator* pin_tedges_invalidator,
                                                   TimingInfo* timing_info) {
     VTR_ASSERT_SAFE(timing_info);
     VTR_ASSERT_SAFE(pin_tedges_invalidator);
 
-    auto& cluster_ctx = g_vpr_ctx.clustering();
-    auto& clb_nlist = cluster_ctx.clb_nlist;
-
-    //Inalidate timing graph edges affected by the move
-    for (ClusterPinId pin : blocks_affected.affected_pins) {
-        //It is possible that some connections may not have changed delay.(e.g.
+    //Invalidate timing graph edges affected by the move
+    for (ClusterPinId clb_pin : sink_pins_affected) {
+        //It is possible that some connections may not have changed delay.
```

Comment:
VB: changed from old code, but I went through this and am pretty sure the net algorithm is the same so I don't think this is the slowdown.